test/puppetlabs/puppetdb/testutils/services_test.clj

```diff
@@ -0,0 +1,23 @@
+(ns puppetlabs.puppetdb.testutils.services-test
+  (:require [clojure.test :refer :all]
+            [clojure.tools.logging :as log]
+            [puppetlabs.puppetdb.testutils.services :as tgt])
+  (:import [org.slf4j Logger LoggerFactory]))
+
+(defn- event->map [event]
+  {:level (str (.getLevel event))
+   :message (.getMessage event)})
+
+(deftest with-log-level-and-logging-to-atom
+  (let [log (atom [])]
+    (tgt/with-log-level Logger/ROOT_LOGGER_NAME :error
+      (tgt/with-logging-to-atom Logger/ROOT_LOGGER_NAME log
+        (log/info "wlta-test"))
+      (is (not-any? #(= {:level "INFO" :message "wlta-test"} %)
+                    (map event->map @log)))))
+  (let [log (atom [])]
+    (tgt/with-log-level Logger/ROOT_LOGGER_NAME :info
+      (tgt/with-logging-to-atom Logger/ROOT_LOGGER_NAME log
+        (log/info "wlta-test"))
+      (is (some #(= {:level "INFO" :message "wlta-test"} %)
+                (map event->map @log))))))
```

Comment:
^ that's wrong. For some reason it doesn't fail when I run it in the repl, but it fails with lein test